packages/hierarchy-browser/package.json

```diff
@@ -0,0 +1,53 @@
+{
+	"name": "@essex-js-toolkit/hierarchy-browser",
+	"version": "1.1.2",
+	"description": "React component to display hierarchical community data",
+	"license": "UNLICENSED",
+	"entry": "src/index.ts",
+	"main": "dist/cjs/index.js",
```

Comment:
"main": "src/index.ts", and remove "module" and "types" from the top-level pkg.json.

We're moving towards using TypeScript source directly during development and then swapping in the dist/ folder on publication, which is what the `publishConfig` does